flake.nix

```diff
@@ -7,11 +7,7 @@
     };
     nixpkgs.follows = "haskellNix/nixpkgs-unstable";
     flake-utils.url = "github:numtide/flake-utils";
-    pre-commit-hooks = {
-      url = "github:cachix/pre-commit-hooks.nix";
-      inputs.nixpkgs.follows = "nixpkgs";
-      inputs.flake-utils.follows = "flake-utils";
-    };
+    pre-commit-hooks.url = "github:cachix/git-hooks.nix";
```

Comment:
Why are the "follows" attributes no longer necessary?

Reply:
The `flake-utils` input is no longer necessary since https://github.com/cachix/git-hooks.nix/pull/446, and the `nixpkgs` input of `haskell.nix` is too old to contain `cabal-gild`, so I now just use the `nixpkgs` input by `git-hooks.nix`.